composer.json

```diff
@@ -26,8 +26,8 @@
   },
   "require": {
     "php": ">=7.4",
-    "typo3/cms-core": "^11.5 || dev-master",
-    "sjbr/static-info-tables": "^11.5"
+    "typo3/cms-core": "^11.5 || ^12.4",
+    "sjbr/static-info-tables": "^12.3"
```

Comment:
As far as I can see, there is no tag `12.3.x` for the base extension `static_info_tables` https://packagist.org/packages/sjbr/static-info-tables

Reply:
Same as in the first comment.

Reply:
I assume, we should use `^11.5 || ^12.4` then, right?

But, also for this, I'd like to wait until the correct version is tagged. I don't know when it will follow and for now, there is no official support ☺️